src/website/app/demos/Dialog/examples/Dialog/composition.js

```diff
@@ -0,0 +1,72 @@
+/* @flow */
+import { Component } from 'react';
+import Button from '../../../../../../library/Button';
+import Dialog from '../../../../../../library/Dialog';
+import Select from '../../../../../../library/Select';
+import { statesData as data } from '../../../Select/components/selectData';
+
+export default {
+  id: 'composition',
+  title: 'Composition',
+  description: `TODO Other Mineral components may be composed in the body of the
```

Comment:
I think I left a TODO here just to make sure this is all we wanted to say about this example.